www-client/surf/surf-2.0.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
```

Comment:
This requires a revbump and dropping keywords to `~`. Are you sure you want to do this? Nothing wrong with EAPI-6.

Reply:
I see no problem with it.